manifests/bundle.yaml

```diff
@@ -64,34 +70,44 @@ spec:
   ports:
   - name: metrics
     port: 9100
+    protocol: TCP
+    targetPort: metrics
   selector:
     app: storageos
+    app.kubernetes.io/component: metrics-exporter
     category: metrics-exporter
 ---
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
   labels:
     app: storageos
+    app.kubernetes.io/component: metrics-exporter
   name: storageos-metrics-exporter
   namespace: storageos
 spec:
   selector:
     matchLabels:
       app: storageos
+      app.kubernetes.io/component: metrics-exporter
   template:
     metadata:
       labels:
         app: storageos
+        app.kubernetes.io/component: metrics-exporter
         category: metrics-exporter
     spec:
       containers:
       - args:
         - -config
         - /etc/storageos/metrics-exporter-config.yaml
-        image: storageos/metrics-exporter:v0.1.5
+        image: storageos/metrics-exporter:v0.1.6
         imagePullPolicy: IfNotPresent
         name: storageos-metrics-exporter
+        ports:
+        - containerPort: 9100
```

Comment:
We have a flag for port number and a hard-coded number here. As I know the user doesn't have any control over the manifests we deploy with the operator. So that means configuration doesn't make much sense, because if the customer somehow magically passes the port number, they have to mount a modified manifest also into the operator container. WDYT?